steps/cli/oc.go

```diff
@@ -168,4 +169,40 @@ var _ = gauge.Step("Configure GitHub token for git resolver in TektonConfig", fu
 var _ = gauge.Step("Configure the bundles resolver", func() {
 	patch_data := "{\"spec\":{\"pipeline\":{\"bundles-resolver-config\":{\"default-kind\":\"task\", \"defaut-service-account\":\"pipelines\"}}}}"
 	oc.UpdateTektonConfig(patch_data)
+})
+
+var _ = gauge.Step("Create signing-secrets for tekton chains", func(){
```

Comment:
I don't understand the logic of this step. IMHO it should be something like
1. check if Chains is already configured, if yes, end
2. else if env variables are specified, use them and create secret
3. else if env variable are not specified, generate new secrets